tools/test/iree-run-module-outputs.mlir

```diff
@@ -2,7 +2,7 @@
 
 // RUN: (iree-compile --iree-hal-target-backends=vmvx %s | \
 // RUN:  iree-run-module --device=local-sync --function=no_output) | \
-// RUN: FileCheck --check-prefix=NO-OUTPUT %s
+// RUN: FileCheck --check-prefix=NO-OUTPUT-LABEL %s
```

Comment:
NO-OUTPUT is prefix, LABEL is directive (https://llvm.org/docs/CommandGuide/FileCheck.html#the-check-label-directive). Without this, the following is not checked.

Reply:
Yes. You're right. I'm closing this PR.